tests/searchcommands/test_generator_command.py

```diff
@@ -59,3 +58,29 @@ def generate(self):
     except ValueError as error:
         assert str(error) == "allow_empty_input cannot be False for Generating Commands"
 
+def test_all_fieldnames_present_for_generated_records():
```

Comment:
This is great, can we add a similar test for `add_field` for streaming commands?

Reply:
Absolutely.
I have added a test for the streaming command.